src/includes/performance-monitoring/configuration/vue.mdx

```diff
@@ -4,39 +4,45 @@ To get started with performance monitoring with Vue.js, first install these pack
 
 ```bash
 # Using yarn
-$ yarn add @sentry/browser @sentry/integrations @sentry/apm
+$ yarn add @sentry/browser @sentry/integrations @sentry/tracing
 
 # Using npm
-$ npm install @sentry/browser @sentry/integrations @sentry/apm
+$ npm install @sentry/browser @sentry/integrations @sentry/tracing
 ```
 
+<Alert level="warning" title="Note"><markdown>
+
+We switched our package name from `@sentry/apm` to `@sentry/tracing`. If you were previously using `@sentry/apm`, please see [Switching to @sentry/tracing](/product/performance/apm-to-tracing/).
+
+</markdown></Alert>
+
 The Vue Tracing Integration allows you to track rendering performance during an initial application load.
 
 Sentry injects a handler inside Vue's `beforeCreate` mixin, providing access to a Vue component during its life cycle stages.
-When Sentry encounters a component named `root`, which is a top-level Vue instance (as in `new Vue({})`), we use our AM Tracing integration,
-and create a new activity named `Vue Application Render`. Once the activity has been created, it will wait until all of its child components render, and there aren't new rendering events triggered within the configured `timeout`, before marking the activity as completed.
+When Sentry encounters a component named `root`, which is a top-level Vue instance (as in `new Vue({})`), we use our AM BrowserTracing integration,
```

Comment:
> and there aren't new rendering events triggered within the configured `timeout`

Did this behavior change?

Reply:
I changed this because I felt that we didn't need to expose activities/`idleTransaction` concepts here.

If we really want to do this, we can do it in `BrowserTracing`.

In addition, with `@sentry/tracing`, the Vue integration now can work without `BrowserTracing` and `idleTransactions`, so referring to activities/`timeout` is not always correct.